javascript_nodejs/5.multiturn-prompts-bot/README.md

```diff
@@ -0,0 +1,44 @@
+This sample shows how to use the prompts classes included in `botbuilder-dialogs`. This bot will ask for the user's name and age, then store the response. It demonstrates a multi-turn dialog flow using a text prompt, a number prompt, and multiple state accessors to store and retrieve values.
```

Comment:
This is scrolling off the side of my 4k montior with 8pt fonts turned on. Consider breaking this into 2 or 3 or 4 lines of text.